visualizer/utils/spritesheet.py

```diff
@@ -12,8 +18,18 @@ def __init__(self, filename):
             print(f"Unable to load spritesheet image: {filename}")
             raise SystemExit(e)
 
-    def image_at(self, rectangle, colorkey=None):
-        """Load a specific image from a specific rectangle."""
+    def image_at(self, rectangle: tuple[int, int, int, int], colorkey: pygame.Color | None = None) -> pygame.Surface:
+        """
+        Load a specific image from a specific location .
+        :param rectangle: A tuple of its representing x,y,width,height
```

Comment:
The param rectangle part is slightly confusing. "... of its representing"

Reply:
Good catch. I missed the "n" in ints